cmd.go

```diff
@@ -76,16 +76,41 @@ func CountFlagBuilder(dst *int, long, short, help string) FlagBuilder {
 	}
 }
 
+type ContraintBuilder func(*cobra.Command)
+
+func OneRequired(flags ...string) ContraintBuilder {
+	return func(cmd *cobra.Command) {
+		cmd.MarkFlagsOneRequired(flags...)
+	}
+}
+
+func RequiredTogether(flags ...string) ContraintBuilder {
+	return func(cmd *cobra.Command) {
+		cmd.MarkFlagsRequiredTogether(flags...)
+	}
+}
+
+func MutuallyExclusive(flags ...string) ContraintBuilder {
+	return func(cmd *cobra.Command) {
+		cmd.MarkFlagsMutuallyExclusive(flags...)
+	}
+}
+
 type Command struct {
-	Cmd   *cobra.Command
-	Flags []FlagBuilder
-	once  *sync.Once
+	Cmd         *cobra.Command
+	Flags       []FlagBuilder
+	once        *sync.Once
+	Constraints []ContraintBuilder
```

Comment:
nittypicky: ContraintBuilder is missing an s -> Con**s**traintBuilder.

Reply:
Ups